### ISSUE TITLE
GHDL warning: no option expected after files

### ISSUE THREAD

**Issue by dbemlogic** [HDLUtils/hdlregression#20]
I'm attempting to compile for GHDL using a compile option for one of the files.

`hr.add_files(com_options="-fsynopsis", filename=<path_to_file>, library_name=<library_name>)`

When I compile I get the following error

```
ghdl:warning: no option expected after files, use ./--workdir=<path_to_lib>/v08 [-Wunexpected-option]
ghdl: cannot open
Error: Program ended with exit code 1
Failed to compile <path_to_file>!
```

Maybe the option is attempted to be set after the file path?

GHDL requires the following syntax: 
`$ ghdl -a [options] files`

**Comment by HDLUtils** [comment]
Hi,
What version of GHDL are you using?

**Comment by dbemlogic** [comment]
GHDL 4.0.0-dev (v3.0.0-558-g8a9eb7e25) [Dunoon edition]
 Compiled with GNAT Version: 9.3.0
 static elaboration, mcode code generator
Written by Tristan Gingold.

**Comment by HDLUtils** [comment]
Hi Daniel.
I will publish a fix soon. Also note that you need to correct the misspelling in your compile option.

Regards,

**Comment by HDLUtils** [comment]
The issue you reported has been addressed and resolved in version 0.49.2
